primitives/src/spender.rs

```diff
@@ -27,23 +28,19 @@ pub struct Aggregate {
 }
 #[cfg(feature = "postgres")]
 mod postgres {
-    use std::convert::TryFrom;
-    use tokio_postgres::{Error, Row};
-
     use super::*;
 
-    impl TryFrom<Row> for Spendable {
-        type Error = Error;
 
-        fn try_from(row: Row) -> Result<Self, Self::Error> {
-            Ok(Spendable {
-                spender: row.try_get("spender")?,
-                channel: row.try_get("channel")?,
+    impl From<Row> for Spendable {
```

Comment:
Could you please remove the `derive(Serialize, Deserialize)` from `Spendable` as we don't use it for responses, we have the `SpendableReponse` for that